extensions/ql-vscode/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## [UNRELEASED]
 
+- Implement syntax highlighting for the `additional` keyword.
```

Comment:
Do you mind including the PR link to the CHANGELOG entries? E.g.

```suggestion
- Implement syntax highlighting for the `additional` keyword. [#2609](https://github.com/github/vscode-codeql/pull/2609)
```

Reply:
I have done that in commit 4.

Reply:
Thanks!